js/script.js

```diff
@@ -0,0 +1,41 @@
+$(window).ready(function () {
+
+	var city;
+	var cityPhotoRegistry = {
+		"New York" : "nyc",
+		"New York City" : "nyc",
+		"NYC" : "nyc",
+		"San Francisco" : "sf",
+		"SF" : "sf",
+		"Bay Area" : "sf",
+		"Los Angeles" : "la",
+		"LA" : "la",
+		"LAX" : "la",
+		"Austin" : "austin",
+		"ATX" : "austin",
+		"Sydney" : "sydney",
+		"SYD" : "sydney"
+	}
+
+	$("#myform").on("submit", function (e) {
+		//stop the browser from refreshing the page
+		e.preventDefault();
+
+		//get user input and trim any unnecessary white space characters
+		city = $("#city-type").val().trim();
```

Comment:
You could also do ```city = $("#city-type").val().trim().toLowerCase();``` and change the object keys to lowercase so that your app is case sensitive.